README.md

```diff
@@ -49,9 +49,8 @@ For more information and distro-specific instructions, see the [Linux installati
 
 | Install:            | Upgrade:            |
 | ------------------- | --------------------|
-| `winget install gh` | `winget install gh` |
+| `winget install gh` | `winget upgrade gh` |
```

Comment:
[Winget release notes](https://github.com/microsoft/winget-cli/releases/tag/v0.2.2941-preview) mention that this feature is "experimental". What does that mean? Does everyone using winget have access to this command? Does it work well, in your experience?

Reply:
I am getting this 
![image](https://user-images.githubusercontent.com/65185487/101134450-62de5480-3630-11eb-84dd-202342a49757.png)

Reply:
@mislav ?

Reply:
It looks like the manifest for gh v1.3.1 is already in the winget repository: https://github.com/microsoft/winget-pkgs/pull/5168

If `upgrade` doesn't work for you, then the command might be buggy still. Can you try `winget install gh`?

Reply:
Yes winget install gh works well and good.

Reply:
But if I write “winget upgrade gh ” it just don’t even show this output “Found Github CLI” so i have to write “winget upgrade ’Github CLI’ ”

Reply:
Thanks for checking. It sounds like `install` works and `upgrade` doesn't. So I'll go ahead and close this until a future release of Winget where this is fixed.

Reply:
Ok would be definitely waiting for that.

Reply:
@mislav I am the @alannt777. I believe the upgrade command works only for apps packaged using MSIX.

Reply:
@itsme-alan what happened to all your repos.

Reply:
OFF-TOPIC: I had to get a new account. I thought I would delete the old one